src/main/java/sky/afk/AfkPlugin.java

```diff
@@ -74,50 +78,64 @@ public void init(){
 
         Timer.schedule(() -> {
             if(state.isPlaying()){
+                //Call.sendMessage("running afk timer");
                 for(Player player : Groups.player){
+                    if (player==null) continue;
```

Comment:
This line doing nothing.
Look https://github.com/Anuken/Mindustry/blob/044fb30b1b7ab26bf9e9ea97c9152ed4d68e08f0/core/src/mindustry/entities/EntityGroup.java#L157

Reply:
i dont understand what you mean.
do u mean the Player player in Groups.player will always be not null?